src/content/pages/build/taco.md

```diff
@@ -0,0 +1,213 @@
+---
+template: build-page/taco
+path: /build/taco
+title: TACo Page
+seoTitle: Threshold Taco
+seoDescription: "TACo (Threshold Access Control) enables end-to-end encrypted data sharing and communication."
+tacoInfo:
+  title: "TACo: Privacy without trust"
+  description: "Give your users true sovereignty over their data, today. TACo – the first and only access control layer you aren’t forced to trust."
+  image: /images/taco-encryption.svg
+  buttons:
+    - label: Start Building
+      url: https://docs.threshold.network/app-development/threshold-access-control-tac/integration-guide
+      variant: EXTERNAL_SOLID
+    - label: Trust Overview
+      url: https://docs.threshold.network/app-development/threshold-access-control-tac/trust-assumptions/cbd-mainnet-version
+      variant: EXTERNAL_OUTLINE
+  rowReverse: false
+tacoFeaturesInfo:
+  title: "The Fully Decentralized End-to-End Encryption"
+  description: "Empower users to share data with qualifying recipients, and no-one else Encryption via a decentralized cryptosystem."
+  highlighted: ["Decentralized", "Encryption"]
+tacoFeatures:
+  - title: "Web3 needs TACo"
+    cards:
+      - title: "Decentralized Storage"
+        description: "TACo is the perfect compliment to persistent storage, finally enabling users to grant future access without having to download & re-encrypt locally."
+        icon: /images/storage-icon.svg
+      - title: "Web3 Social Networks"
+        description: "TACo enables many-to-many data sharing, minus a centralized  authority that eventually decides to harvest and monetize interpersonal communication."
```

Comment:
```suggestion
        description: "TACo enables many-to-many data sharing, minus a centralized authority that may eventually decide to harvest and monetize interpersonal communication."
```

Reply:
Fine with either! Was loosely referencing the 'Don't Become Evil' component of d/acc